### PR TITLE
Docker compose

### PR DESCRIPTION
**Pull request by Tristramg**
partial_fix etalab/transport-site#184

### CODE REVIEW

File: datatools/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+RUN apt-get update \
+    && apt-get --yes upgrade \
+    && apt-get --yes install git maven
+RUN git clone -b fix_graphql_spec https://github.com/l-vincent-l/datatools-server.git /app \
+ && cd /app && mvn -Dmaven.test.skip=true package
+
+COPY *.yml /app/configuration/
+
+ENTRYPOINT [ "java", "-jar", "/app/target/dt-57520e3.jar", "/app/configuration/env.yml", "/app/configuration/server.yml"]
```
Comment (maukoquiroga) [inline datatools/Dockerfile]: Won't that target change over time ?

Comment (maukoquiroga) [reply datatools/Dockerfile]: You could use something like ```git rev-parse --short HEAD``` or I don't know how to interpolate with Docker.

**Comment by Tristramg** [comment]
Maybe the cleanner way would be to do a `mv /app/target/dt*jar
/app/dt.jar`? What do you think ?

**Comment by maukoquiroga** [comment]
Sounds good!

**Comment by Tristramg** [comment]
Changed

**Comment by maukoquiroga** [comment]
Cannot make it work:

capture d ecran 2018-03-12 a 16 53 49

Also, to be more precise, I suggest we add something like this to the ```README```:

```
docker-machine create datatools
docker-machine start datatools
docker-compose up datatools
```

**Comment by Tristramg** [comment]
This is an :chicken: :egg: situation. I wanted to build the image on dockerhub, but it can’t build it as long as the Dockerfile is not provided :/ let’s talk about it when I’ll be at work

`docker-machine` is a command specific to macOS, isn’t it ?

**Comment by Tristramg** [comment]
* I pushed the image on dockerhub, it should work now
* The readme has been updated
* I changed the Dockerfile to reduce the size from 4xx Mb down to 220 Mb

**Pull request merged**